modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -990,24 +990,25 @@ When defining an output of type JSON, there are a few optional parameters you ca
 |`encoding`| string | UTF-8 |The character set to be used for the output
 |`bufferSize`| number | 153600 | The size of the buffer writer
 |`inlineCloseOn` | string | | When the writer should use inline close tag. Possible values = empty/none
-|`skipNullOn`| string | | Possible values = `elements`/`attributes`/`everywhere`. See <<Skip Null On>>
+|`skipNullOn`| string | | Possible values = `objects`/`arrays`/`everywhere`. See <<skip_null_on_json>>.
 |`duplicateKeyAsArray`| boolean | false | JSON language doesn't allow duplicate keys with one same parent, this usually raises an exception. If set to true, the output contains a single key that points to an array containing all the values assigned to it.
 |===
 
 .Example: output Directive
 [source,dataweave,linenums]
 ----
-output application/json indent=false, skipNullOn="arrays"
+output application/json indent=false, skipNullOn="everywhere"
 ----
 
+[[skip_null_on_json]]
 ==== Skip Null On (for JSON)
 
 You can specify whether this generates an outbound message that contains fields with "null" values, or if these fields are ignored entirely. This can be set through an attribute in the output directive named `skipNullOn`, which can be set to three different values: `elements`, `attributes`, or `everywhere`.
```

Comment:
```suggestion
You can specify whether this generates an outbound message that contains fields with "null" values, or if these fields are ignored entirely. This can be set through an attribute in the output directive named `skipNullOn`, which can be set to three different values: `objects`, `arrays`, or `everywhere`.
```